packages/kbn-grid-layout/grid/use_grid_layout_events.ts

```diff
@@ -224,16 +229,31 @@ export const useGridLayoutEvents = ({
     const onMouseMove = (e: MouseEvent) => {
       // Note: When an item is being interacted with, `mousemove` events continue to be fired, even when the
       // mouse moves out of the window (i.e. when a panel is being dragged around outside the window).
-      mouseClientPosition.current = { x: e.clientX, y: e.clientY };
+      pointerClientPosition.current = { x: e.clientX, y: e.clientY };
       calculateUserEvent(e);
     };
 
+    const onTouchMove = (e: TouchEvent) => {
+      if (!interactionEvent$.value) {
+        return;
+      }
+      if (e.touches.length > 1) return;
+      const touch = e.touches[0];
+      pointerClientPosition.current = { x: touch.clientX, y: touch.clientY };
+      e.preventDefault();
+      e.stopPropagation();
+      // `shouldAutoScroll` is set to false because we don't want the screen to scroll when dragging/resizing the items
+      calculateUserEvent(e, false);
```

Comment:
Hmmm... I'm still getting some strange scrolling at times - not really sure what is causing it, but it's like the panel gets stuck 🤔 

https://github.com/user-attachments/assets/9b2b7123-6020-468a-a49e-92eff5264988